server/routerlicious/packages/services/src/mongodb.ts

```diff
@@ -447,6 +447,8 @@ export class MongoDb implements core.IDb {
 	}
 }
 
+export type ConnectionNotAvailableMode = "notRetry" | "stop"; // Ideally we should have 'delayRetry' options, but that requires more refactor on our retry engine so hold for this mode;
```

Comment:
>notRetry [](http://example.com/codeflow?start=42&length=8)

(nit) `dontRetry`?

Reply:
Where is it used?

Reply:
Changed to `ruleBehavior` to indicate the code should follow the original rule behavior.

This is a setting as `default` if we don't specify a setting
![image](https://user-images.githubusercontent.com/3719246/235157226-aaf1138c-7a46-4e3d-8ded-c25f2e93bc4d.png)